packages/editor/src/components/post-panel-row/style.scss

```diff
@@ -1,13 +1,19 @@
 .editor-post-panel__row {
 	width: 100%;
-	justify-content: flex-start;
-	align-items: flex-start;
 	min-height: $button-size;
 }
 
+.editor-post-panel__row.has-label {
+	justify-content: flex-start !important;
+	align-items: flex-start !important;
```

Comment:
Maybe explain why this needs to be conditional now?

Reply:
Good remark, the conditional style is actually not needed now.